maths/simpson_rule.py

```diff
@@ -12,6 +12,19 @@
 def method_2(boundary, steps):
```

Comment:
Please add type hints.
```suggestion
def method_2(boundary: list[int], steps: int) -> float:
```